web/pdf_find_controller.js

```diff
@@ -322,10 +322,9 @@ var PDFFindController = (function PDFFindControllerClosure() {
      * @param {number} index - match index.
      * @param {Array} elements - text layer div elements array.
      * @param {number} beginIdx - start index of the div array for the match.
-     * @param {number} endIdx - end index of the div array for the match.
```

Comment:
Since there is currently no support for that, I would say we remove the unused parameter for now. Once there is support for that, we can re-add it and actually use it in this method.